src/tests/system/tests/test_passkey.py

```diff
@@ -26,6 +26,23 @@ def passkey_requires_root(client: Client) -> tuple[bool, str] | bool:
     return True
 
 
+@pytest.fixture()
+def umockdev_ipaotpd_update(mh):
+    """
+    Update the ipa-optd@.service file from ipa server
+    for umockdev
+
+    :return: True or False
+    :rtype: bool
+    """
+    ipa: IPA = mh.fixtures["ipa"]
+    ipa_otpd = "/usr/lib/systemd/system/ipa-otpd@.service"
+    content = "Environment=LD_PRELOAD=/opt/random.so"
+    ipa.fs.append(ipa_otpd, content)
+    for cmd in ["systemctl daemon-reload", "ipactl restart"]:
+        ipa.host.ssh.run(cmd)
```

Comment:
This can be made a two liner:

```python
ipa.fs.append(""/usr/lib/systemd/system/ipa-otpd@.service", "Environment=LD_PRELOAD=/opt/random.so")
ipa.svc.restart("ipa")
```

Reply:
updated.